src/hint/specifications/hint.md

```diff
@@ -0,0 +1,33 @@
+# Hint
+
+Use it to show the member there’s a new action they can take on this page. Only use for actions that previously didn’t exist on this page.
```

Comment:
It would be good to have something also describing when the action is **not considered as "new" anymore**. Then maybe we could even programaticaly shows/hide the component during a certain period of time. Or just making sure that we remove it when it's not "new" anymore. I'll talk to Jonas :)

Reply:
That's a good point, I should also talk about it with Alban. Because it might stay new for several months with the situation (half trolling). But we will forget it for sure.

Reply:
We could implement an expirationDate that would hide the component and log a warning for the devs.

Reply:
Or implement a proper user backend preference/flag store :) ? The expiration date strategy makes assumption about the user visit pattern and is some client business logic (spread in 3 clients).

Reply:
> implement a proper user backend preference/flag store
This should be driven by the product team depending on what they want to have:
- is the hint component used on a user-centric point of view? e.g. This new action will be considered new until the user sees it 3 times, need to be in the backend.
- is the hint component used on a product point of view only? e.g. This action is considered new in the product for X months (regardless of the user), no need for backend.